src/Essentials/src/Geolocation/Geolocation.ios.macos.cs

```diff
@@ -116,4 +192,36 @@ public override void LocationsUpdated(CLLocationManager manager, CLLocation[] lo
 
 		public override bool ShouldDisplayHeadingCalibration(CLLocationManager manager) => false;
 	}
+
+	class ContinuousLocationListener : CLLocationManagerDelegate
+	{
+		internal Action<CLLocation> LocationHandler { get; set; }
+
+		internal Action<GeolocationError> ErrorHandler { get; set; }
+
+		public override void LocationsUpdated(CLLocationManager manager, CLLocation[] locations)
+		{
+			var location = locations?.LastOrDefault();
+
+			if (location == null)
+				return;
+
+			LocationHandler?.Invoke(location);
+		}
+
+		public override void Failed(CLLocationManager manager, NSError error)
```

Comment:
Code documentation, see earlier comments.

Since this is an override you can probably just do `/// <inheritdoc/>`